src/Airship/Internal/Decision.hs

```diff
@@ -242,8 +242,11 @@ b04 r@Resource{..} = do
 b03 r@Resource{..} = do
     trace "b03"
     req <- lift request
+    allowed <- lift allowedMethods
     if requestMethod req == HTTP.methodOptions
-        then lift $ halt HTTP.status200
+        then do
+            lift $ addResponseHeader ("Allow",  intercalate "," allowed)
+            lift $ halt HTTP.status200
```

Comment:
> Maybe 204 is best unless we also want to change OPTIONS requests to return a body as the cnn.com request does.

I agree. I use 204 to signal my intention that the client doesn't even have to test the body because it's going to be empty.

There are lots of "RESTful apps" (note the air quotes) that return 200 for *every* successful response, rather than a more precise status like 201 or 204. I wouldn't necessarily give much weight to in-the-wild behaviour compared to the intended semantics specified in the RFCs.